src/test/java/org/takes/rs/RsXSLTTest.java

```diff
@@ -125,6 +126,46 @@ public Source resolve(final String href,
     }
 
     /**
+     * RsXSLT closes decorated Response body's InputStream when XML conversion
+     * is done.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closesDecoratedResponseInputStream() throws IOException {
```

Comment:
@Happy-Neko let's declare throws `Exception` here. See [this article](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names).